modules/Hoot/tools/folderTree.js

```diff
@@ -909,4 +916,9 @@ export default class FolderTree extends EventEmitter {
 
         this.update( d );
     }
+
+    setDoubleClickHandler(ref, handler) {
+        this.doubleClickRef = ref;
+        this.doubleClickHandler = handler;
```

Comment:
this feels kinda hacky, but not sure how else to call `submitLayer` from `folderTree`